src/routes/discover/network/[network]/+page.svelte

```diff
@@ -1,45 +1,85 @@
 <script lang="ts">
-	import { getTmdbNetworkSeries } from '$lib/apis/tmdb/tmdbApi';
+	import { getTmdbNetworkSeriesQuery, type DiscoverTvQuery } from '$lib/apis/tmdb/tmdbApi';
 	import Card from '$lib/components/Card/Card.svelte';
-	import CardGrid from '$lib/components/Card/CardGrid.svelte';
 	import CardPlaceholder from '$lib/components/Card/CardPlaceholder.svelte';
 	import { fetchCardTmdbSeriesProps } from '$lib/components/Card/card';
 	import type { ComponentProps } from 'svelte';
 	import type { PageData } from './$types';
 	import GridPage from '$lib/components/GridPage/GridPage.svelte';
-	import { networks, type Network } from '../../../../lib/discover';
+	import { networks, type Network } from '$lib/discover';
+	import InfiniteScroll from '$lib/components/GridPage/InfiniteScroll.svelte';
 
 	export let data: PageData;
 
 	const network = networks[data.network] || undefined;
 
-	async function fetchNetworkSeries(network: Network) {
-		const shows = await getTmdbNetworkSeries(network.tmdbNetworkId);
+	let currentPage: number = 1;
```

Comment:
You can omit the types when a value is immediately assigned btw, TypeScript will infer the type correctly